packages/apps/composer-app/src/main.tsx

```diff
@@ -123,8 +123,8 @@ const main = async () => {
     ),
     placeholder: (
       <ThemeProvider tx={defaultTx}>
-        <div className='flex bs-[100dvh] justify-center items-center'>
-          <Status indeterminate aria-label='Initializing' />
+        <div className='flex flex-col justify-end bs-[100dvh]'>
```

Comment:
Does this just put `Status` on the bottom? Does that not look quite empty?

Reply:
let's try it and review.

Reply:
Please summarize all other changes in the description and mark this PR as `feat` or `fix` if it includes noticeable changes in the design. Screenshots & screen captures would also be very helpful when reviewing PRs like this.